README.md

```diff
@@ -41,39 +41,41 @@ This tool expects to run in an environment with the following attributes:
 
 Requesting a code review:
 
-    git appraise request
+    git review request
 
 Pushing code reviews to a remote:
 
-    git appraise push [<remote>]
+    git review push [<remote>]
 
 Pulling code reviews from a remote:
 
-    git appraise pull [<remote>]
+    git review pull [<remote>]
 
 Listing open code reviews:
 
-    git appraise list
+    git review
+    git review --list
 
 Showing the status of the current review, including comments:
 
-    git appraise show
+    git review show
 
 Showing the diff of a review:
 
-    git appraise show --diff [--diff-opts "<diff-options>"] [<review-hash>]
+    git review show -p/-u/--patch [--diff-opts "<diff-options>"] [<review-hash>]
```

Comment:
I'm going to have to mull this over a bit more.

Using the "show" command to print the diff is probably one of the least polished parts of the command line surface.

If we go with the -p/-u/--path flag, then we should also add a -s/--no-patch and make the default configurable. I'd also want to keep the --diff option so as to not break any scripts that have been written against the tool. Right now I'm leaning towards doing that, but I don't want to make a hasty decision about it.

Another thing I've wondered about is whether or not diff should be promoted to its own subcommand (e.g. "git appraise diff"). I can't decide whether or not that strikes the right balance between "easy-to-use" and "small-CLI-surface".

I'll split this off into its own tracking issue.